docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -3298,6 +3298,19 @@ aaa group server radius rad_mac
 aaa authentication login mac_methods group rad_mac
 ----
 
+Aironet 1600
+^^^^^^^^^^^^
+
+CoA and radius:
+----
+radius-server attribute 32 include-in-access-req format %h
+radius-server vsa send accounting
+aaa server radius dynamic-author
+ client 192.168.0.10
```

Comment:
Is that supposed to be the PacketFence server ? If yes, the guide assumption is '192.168.1.5'